clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2030,6 +2040,15 @@ class SyclKernelArgsSizeChecker : public SyclKernelFieldHandler {
   using SyclKernelFieldHandler::handleSyclHalfType;
 };
 
+static const CXXMethodDecl *getOperatorParens(const CXXRecordDecl *Rec) {
+  for (const auto *D : Rec->decls()) {
+    if (const CXXMethodDecl *MD = dyn_cast<CXXMethodDecl>(D))
```

Comment:
I think iterating through methods will be more optimal
```suggestion
  for (const auto *MD : Rec->methods()) {
```

Reply:
ok